src/components/EditorWidgets/Date/DateControl.js

```diff
@@ -67,7 +79,7 @@ export default class DateControl extends React.Component {
 
   render() {
     const { includeTime, value, classNameWrapper, setActiveStyle, setInactiveStyle } = this.props;
-    const format = this.format;
+    const format = this.formatDisplay;
```

Comment:
Not sure if we need this since it doesn't actually affect the display, just the parsing.

Reply:
Agreed, updated.